uk.co.compucorp.civicrm.civihrcore/CRM/Smarty/plugins/function.civihrVersion.php

```diff
@@ -48,7 +48,7 @@
  * @return string
  */
 function smarty_function_civihrVersion($params, &$smarty) {
-  $result = civicrm_api3('CiviHR', 'getversion');
+  $result = civicrm_api3('CiviHRCore', 'getversion');
```

Comment:
since you are now inside the same extension , it would probably make more sense ( and faster) to call the method directly rather than using the APi